DependencyInjection/Configuration.php

```diff
@@ -30,6 +30,10 @@ public function getConfigTreeBuilder()
                     ->info('The priority of the event listener that converts an Exception to a Response')
                     ->defaultValue(10)
                 ->end()
+                ->scalarNode('custom_role')
```

Comment:
@rjwebdev Keep the naming consistent throughout the code. I personally find `role_prefix` to be more descriptive. :)

Reply:
Changed